hindsight/modules/heka/elasticsearch.lua

```diff
@@ -0,0 +1,120 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
not needed, the sandbox contains it 
http://mozilla-services.github.io/lua_sandbox/modules/heka/elasticsearch.html